configuration.rst

```diff
@@ -131,6 +133,21 @@ db-extra-search-path
 
   Multiple schemas can be added in a comma-separated string, e.g. ``public, extensions``.
 
+.. _db-channel:
+
+db-channel
+----------
+
+  The name of the notification channel that PostgREST uses for :ref:`schema_reloading` and configuration reloading.
+
+.. _db-channel-enabled:
+
+db-channel-enabled
+------------------
+
+  When this is set to :code:`true`, the notification channel specified in :ref:`db-channel` is enabled.
+  Note that this setting is not compatible with connection poolers like PgBouncer in transaction pooling mode.
```

Comment:
I think we should remove this for now, since we're not really compatible with pgbouncer.
```suggestion
```